source/apiVolontaria/volunteer/models.py

```diff
@@ -1,3 +1,25 @@
 from django.db import models
 
-# Create your models here.
+
+class TaskType(models.Model):
+
+    """
+
+    This class represents the TaskType model.
+
+    """
```

Comment:
Why do you put blank line in your comments all the time ?
It take a lot of lines and i never see that before

Reply:
I took the idea from [this blog]() which summarized PEP257. Here is the concerned part:
> class Documentation(object):
>
>   """Illustrate class-level docstring.
>
>    Classes use a special whitespace convention: the opening and closing quotes
    are preceded and followed by a blank line, respectively. No other
    docstrings should be preceded or followed by anything but code.
>
>    A blank line at the end of a multi-line docstring before the closing
    quotation marks simply makes it easier for tooling to auto-format
    paragraphs (wrapping them at 79 characters, per PEP8), without the closing
    quotation marks interfering. For example, in Vim, you can use `gqip` to
    "apply text formatting inside the paragraph." In Emacs, the equivalent
    would be the `fill-paragraph` command. While it's not required, the
    presence of a blank line is quite common and much appreciated. Regardless,
    the closing quotation marks should certainly be on a line by themselves.
>
>    """

Maybe the newlines aren't really necessary in our case. We should establish a docstring convention of our own as soon as possible.

Reply:
The link to the blog doesn't work (it's a link to this same page).

It's interessting, i've saw lot of people do some comment like this :

```
"""My first line
rest of
my content
"""
```

But to be more easy to manage with PEP8 i always wrote like this :

```
"""
My first line
rest of
my content
"""
```

Like it show in the [PEP257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) you quoted.

I've open a new issues on this point => #36

Reply:
Great! Yep, generally the first line is a short sentence describing the content of the whole docstring.
> """Summary of the content
>
>Multiple line 
>of content
>
>"""

Thanks for the issue!

PS: Fixed the [link](http://blog.dolphm.com/pep257-good-python-docstrings-by-example/)